theia-rust-docker/next.package.json

```diff
@@ -41,6 +41,7 @@
         "@theia/userstorage": "next",
         "@theia/variable-resolver": "next",
         "@theia/workspace": "next",
+        "@theia/vsx-registry": "next",
```

Comment:
It seems low-risk to keep this change in this untested image, so I am ok to leave this as-is

Reply:
I built the rust image locally without any issues so the vsx-registry does not cause any issues. I guess the problem on travis was related to system performance which was causing the tests to take longer than 30s. I didn't investigate into this issue though.

Reply:
> I built the rust image locally without any issues so the vsx-registry does not cause any issues. I guess the problem on travis was related to system performance which was causing the tests to take longer than 30s. I didn't investigate into this issue though.

I think I found the issue: we are still including a deprecated extension for this image: `@theia/editorconfig`. Long story short, this causes the app never to start, which eventually fails CI. 

I'm removing this extension in this WIP PR: https://github.com/theia-ide/theia-apps/pull/343